Competition2017/src/competition/subsystems/fuel_launcher/FuelLauncherSubsystem.java

```diff
@@ -0,0 +1,185 @@
+package competition.subsystems.fuel_launcher;
+
+import org.apache.log4j.Logger;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+
+import edu.wpi.first.wpilibj.Timer;
+
+import com.ctre.CANTalon.FeedbackDevice;
+import com.ctre.CANTalon.TalonControlMode;
+import xbot.common.command.BaseSubsystem;
+import xbot.common.controls.actuators.XCANTalon;
+import xbot.common.controls.sensors.XTimer;
+import xbot.common.injection.wpi_factories.WPIFactory;
+import xbot.common.properties.BooleanProperty;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+
+@Singleton
+public class FuelLauncherSubsystem extends BaseSubsystem {
+    private static Logger log = Logger.getLogger(FuelLauncherSubsystem.class);
+
+    private final XCANTalon masterMotor;
+    //private final XCANTalon slaveMotor;
+    
+    // output telemetry properties
+    private final DoubleProperty shooterCurrentSpeed;
+    private final DoubleProperty shooterTargetSpeed;
+    private final DoubleProperty shooterOutputPower;
+    private final DoubleProperty shooterTalonError;
+    private final BooleanProperty atSpeedProp;
+    
+    //private final DoubleProperty encoderCodesProperty;
+    private final DoubleProperty p;
+    private final DoubleProperty i;
+    private final DoubleProperty d;
+    private final DoubleProperty f;
+    
+    private final DoubleProperty shooterSpeedThresh;
+
+    @Inject
+    public FuelLauncherSubsystem(WPIFactory factory, XPropertyManager propManager) {
+        log.info("Creating FuelLauncherSubsystem");
+        
+        //encoderCodesProperty = propManager.createPersistentProperty("Launcher Encoder TPC", 4096);
+
+        p = propManager.createPersistentProperty("FuelLauncher wheel P", 0);
+        i = propManager.createPersistentProperty("FuelLauncher wheel I", 0);
+        d = propManager.createPersistentProperty("FuelLauncher wheel D", 0);
+        f = propManager.createPersistentProperty("FuelLauncher wheel F", 0);
+        
+        // Reading through the manual, it looks like only a few things support human units. It might be easier to just keep everything in native units per 100ms,
+        // which is what the Talon uses for all its calculations.
+        // I call this Ticks per Centisecond, or TPC
```

Comment:
Yes, this is a typo - should be deciseconds (since it's in 100ms increments, which is 1/10th of a second)